CHANGELOG.md

```diff
@@ -12,7 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Hypothesis strategies for acquisition functions
 - `Kernel` base class allowing to specify kernels
 - `MaternKernel` class can be chosen for GP surrogates
-- `hypothesis` strategies and roundtrip test for kernels, constraints and objectives
+- `GammaPrior` can now be chosen as lengthscale prior
```

Comment:
@AdrianSosic : Just a ping that rebasing is done. Note that I was quite radical in trimming down the commits, but I have a backup in hand in case you'd prefer more commits.

Reply:
Actually disagree here. If you already see an issue here, why wait? I also already pushed a draft version of the scale kernel approach, so please have a look there, because that is (besides the fact that it is not yet ready to merge due to this not yet being merged) already ready for review (put still a bit messy due to it being based on this)